azext_iot/central/params.py

```diff
@@ -106,12 +138,14 @@ def load_central_arguments(self, _):
         context.argument(
             "interface_id",
             options_list=["--interface-id", "-i"],
-            help="Interface name as specified in the device template. Example: c2dTestingTemplate_356",
+            help="The name of the interface as specified in the device template. You can find it by navigating to Device"
+            " Template and view the interface identity under the corresponding device capability.",
         )
         context.argument(
             "command_name",
             options_list=["--command-name", "--cn"],
-            help="Command name as specified in device template. Example: run_firmware_update",
+            help="The command name as specified in the device template. Command name could be different from the Display"
```

Comment:
we should add a doc/link to a doc on how to get this

Reply:
Will work with Pm to address later.